content/en/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -0,0 +1,99 @@
+
+> We are going to start seeing some awareness about moving away from the words. Moving away from just putting the source into the target language and just pure translation to start having more questions about how the text might be perceived in China, in Russia, or in Korea.
+
+King seems to be rather ahead of the curve &mdash; during the live presentation, an audience member asked Sepulveda whether King had noticed any relationship between  investment in globalization and the company's bottom line. The answer was a resounding "Yes!" Clearly for King, creative localization has been good for business.
+
+
+## The Future of Localization
+
+What does all mean for app developers who are hoping to release software to geographically distributed audiences? It seems that there are several key challenges that such organizations will have to tackle; from the [technical challenges of distributed systems](https://rotational.io/blog/a-distributed-systems-maze/) to the recognition that English-only may not be a sufficient sales medium &mdash; and perhaps even more importantly, to learning how  rich historic contexts and cultural experiences can inform the buying preferences of potential customers around the globe.
+
+## References
+
+- David Sedaris (2000) [Me Talk Pretty One Day](https://en.wikipedia.org/wiki/Me_Talk_Pretty_One_Day)
+- Donald DePalma, Benjamin Sargent, and Renato Beninatto (2006) [Can't Read, Won't Buy](https://insights.csa-research.com/reportaction/305013126/Marketing)
+- Gregory Warner (2021) ["How to Speak Bad English"](https://www.npr.org/2021/04/21/989477444/how-to-speak-bad-english)
+- Heather Hansen (2018) [2 Billion Voices: How to speak bad English perfectly](https://www.ted.com/talks/heather_hansen_2_billion_voices_how_to_speak_bad_english_perfectly)
+- King (2021) [Virtual event - A Kingdom Exclusive - Live Series with Chen and Miguel!](https://community.king.com/en/blog/discussion/329999/virtual-event-a-kingdom-exclusive-live-series-with-chen-and-miguel)
+- Inlingo (2019) [An Interview with Global Localization Manager at King, Miguel Sepulveda](https://blog.inlingogames.com/en/2019/10/29/the-game-has-to-be-super-amazing-for-you-to-forget-about-the-language-barrier-an-interview-with-global-localization-manager-at-king-miguel-sepulveda/)
+- Inlingo (2021) [What is Localization Testing and Why is it Important?](https://blog.inlingogames.com/en/2021/01/13/what-is-localization-testing-and-why-is-it-important/)
+- Matilda Battersby (2015) [Inside Out: Pixar makes crucial change for Japanese audiences by editing out broccoli](https://www.independent.co.uk/arts-entertainment/films/news/inside-out-pixar-makes-crucial-change-japanese-tastes-editing-out-broccoli-10413794.html)
```

Comment:
I removed the HTML comment at the end with the image reference -- it was causing a blockquote to render with nothing in it at the end of the post - not sure why, it should have stayed a comment.

Reply:
bummer